autotest/pymod/gdaltest.py

```diff
@@ -1534,53 +1516,35 @@ def reregister_all_jpeg2000_drivers():
     global jp2kak_drv, jpeg2000_drv, jp2ecw_drv, jp2mrsid_drv, jp2openjpeg_drv, jp2lura_drv
     global jp2kak_drv_unregistered, jpeg2000_drv_unregistered, jp2ecw_drv_unregistered, jp2mrsid_drv_unregistered, jp2openjpeg_drv_unregistered, jp2lura_drv_unregistered
 
-    try:
-        if jp2kak_drv_unregistered:
-            jp2kak_drv.Register()
-            jp2kak_drv_unregistered = False
-            gdal.Debug('gdaltest', 'Registering JP2KAK')
-    except:
-        pass
-
-    try:
-        if jpeg2000_drv_unregistered:
-            jpeg2000_drv.Register()
-            jpeg2000_drv_unregistered = False
-            gdal.Debug('gdaltest', 'Registering JPEG2000')
-    except:
-        pass
-
-    try:
-        if jp2ecw_drv_unregistered:
-            jp2ecw_drv.Register()
-            jp2ecw_drv_unregistered = False
-            gdal.Debug('gdaltest', 'Registering JP2ECW')
-    except:
-        pass
-
-    try:
-        if jp2mrsid_drv_unregistered:
-            jp2mrsid_drv.Register()
-            jp2mrsid_drv_unregistered = False
-            gdal.Debug('gdaltest', 'Registering JP2MrSID')
-    except:
-        pass
-
-    try:
-        if jp2openjpeg_drv_unregistered:
-            jp2openjpeg_drv.Register()
-            jp2openjpeg_drv = False
-            gdal.Debug('gdaltest', 'Registering JP2OpenJPEG')
-    except:
-        pass
-
-    try:
-        if jp2lura_drv_unregistered:
-            jp2lura_drv.Register()
-            jp2lura_drv = False
-            gdal.Debug('gdaltest', 'Registering JP2Lura')
-    except:
-        pass
+    if jp2kak_drv_unregistered:
+        jp2kak_drv.Register()
+        jp2kak_drv_unregistered = False
+        gdal.Debug('gdaltest', 'Registering JP2KAK')
+
+    if jpeg2000_drv_unregistered:
+        jpeg2000_drv.Register()
+        jpeg2000_drv_unregistered = False
+        gdal.Debug('gdaltest', 'Registering JPEG2000')
+
+    if jp2ecw_drv_unregistered:
+        jp2ecw_drv.Register()
+        jp2ecw_drv_unregistered = False
+        gdal.Debug('gdaltest', 'Registering JP2ECW')
+
+    if jp2mrsid_drv_unregistered:
+        jp2mrsid_drv.Register()
+        jp2mrsid_drv_unregistered = False
+        gdal.Debug('gdaltest', 'Registering JP2MrSID')
+
+    if jp2openjpeg_drv_unregistered:
+        jp2openjpeg_drv.Register()
+        jp2openjpeg_drv = False
+        gdal.Debug('gdaltest', 'Registering JP2OpenJPEG')
+
+    if jp2lura_drv_unregistered:
+        jp2lura_drv.Register()
+        jp2lura_drv = False
```

Comment:
should be jp2lura_drv_unregistered = False

Reply:
Ditto here.